clients/admin-ui/src/features/config-wizard/AuthenticateAwsForm.tsx

```diff
@@ -110,101 +128,193 @@ const AuthenticateAwsForm = () => {
-                          data.
-                          <AccordionButton
-                            display="inline"
-                            padding="0px"
-                            ml="5px"
-                            width="auto"
-                            color="complimentary.500"
-                          >
-                            {isExpanded ? `(show less)` : `(show more)`}
-                          </AccordionButton>
-                        </h2>
-                        <AccordionPanel padding="0px" mt="20px">
-                          In order to run the scanner, please provide
-                          credentials for authenticating to AWS. Please note,
-                          the credentials must have the{" "}
-                          <DocsLink href={DOCS_URL_AWS_PERMISSIONS}>
-                            minimum permissions listed in the support
-                            documentation here
-                          </DocsLink>
-                          . You can{" "}
-                          <DocsLink href={DOCS_URL_IAM_POLICY}>
-                            copy the sample IAM policy here
-                          </DocsLink>
-                          .
-                        </AccordionPanel>
-                      </>
-                    )}
-                  </AccordionItem>
-                </Accordion>
-              </>
-            ) : (
-              <ScannerError error={scannerError} />
-            )}
-
-            <Stack>
-              <CustomTextInput
-                name="aws_access_key_id"
-                label="Access Key ID"
-                // TODO(#724): These fields should link to the AWS docs, but that requires HTML
-                // content instead of just a string label. The message would be:
-                // "You can find more information about creating access keys and secrets on AWS docs here."
-                tooltip="AWS Access Key ID is the AWS ID associated with the account you want to use for scanning."
-              />
-              <CustomTextInput
-                type="password"
-                name="aws_secret_access_key"
-                label="Secret"
-                // "You can find more about creating access keys and secrets on AWS docs here."
-                tooltip="The secret access key is generated when you create your new access key ID."
-              />
-              <CustomSelect
-                name="region_name"
-                label="Default Region"
-                // "You can learn more about regions in AWS docs here."
-                tooltip="Specify the default region in which your infrastructure is located. This is necessary for successful scanning."
-                options={AWS_REGION_OPTIONS}
-              />
-            </Stack>
-            <HStack>
-              <Button variant="outline" onClick={handleCancel}>
-                Cancel
+    <>
+      <AlertDialog
```

Comment:
Nit: Usually elements like this that are isolated from the main content a component displays are put last. Makes it a little easier to follow the flow.